community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -1085,6 +1085,7 @@ public synchronized void stop()
         // Here we're zooming in and focusing on getting committed transactions to close.
         awaitAllTransactionsClosed();
         LogFile logFile = transactionLogModule.logFile();
+        life.stop();
```

Comment:
OK, but please document this fact, both in NeoStoreDataStore and in PhysicalLogFile#shutdown(). Makes sense?